Content.Server/Implants/SubdermalImplantSystem.cs

```diff
@@ -131,6 +134,7 @@ private void OnScramImplant(EntityUid uid, SubdermalImplantComponent component,
                 break;
         }
         _xform.SetWorldPosition(ent, targetCoords.Position);
+        _xform.SetParent(ent, targetParent.IsValid() ? targetParent : _mapManager.GetMapEntityId(xform.MapID));
```

Comment:
you can probably just use AttachToGridOrMap

Reply:
Aye, that works, I wasn't aware the method exists.